test.py

```diff
@@ -582,6 +631,12 @@ def main():
         logger.error("Aborting run.")
         return False
 
+    try:
+        setup_junit_xml()
+    except Exception as e:
+        logging.exception("Failed to load junit_xml; no templated success/fail"
```

Comment:
I don't understand the question.
logging.exception() is special magic that takes an informative string and records the string and the current exception into the logger channel.  I'd have preferred `logger.exception(e, msg)`, but I didn't write it.